network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -418,6 +433,15 @@ class BanneredCampaignPage(PrimaryPage):
 
     show_in_menus_default = True
 
+    def get_intro(self):
+        if self.use_intro_from_parent:
+            parent = self.get_parent()
+            try:
+                return parent.specific.intro
+            except AttributeError:
+                return None
```

Comment:
**Kudos** Nicely done, handling both cases (no parent, and a parent with no intro) in one 👌

Reply:
Yas yay! Thanks! I tried to keep it simple as possible haha